### PR TITLE
[CORE-2143] Update readme for GA

### PR DESCRIPTION
**Pull request by alexdicianu**

### CODE REVIEW

**Comment by mbaynton** [comment]
Over in https://pantheon.slack.com/archives/CM9E09YDS/p1602869411003300 I was advocating for making one final change to these repos before removing the experimental label. In theory we should stop changing files outside of the `upstream-configuration` folder once we declare these repos ready.

**Comment by alexdicianu** [comment]
@mbaynton looks like that card you referenced in slack got deprioritized. I ported some of the documentation updates from https://github.com/pantheon-systems/documentation/pull/5663.

**Comment by mbaynton** [comment]
I think we should keep the documentation pertinent to upstream maintainers in the separate file. Thinking about this being forked to an org's upstream with a bunch of downstream sites, the README for a given downstream site and the README for the upstream become unrelated documents. If we keep it as one file, downstream site maintainers might make edits and upstream maintainers might also make edits, causing merge conflicts just cause of docs.

**Comment by mbaynton** [comment]
I am wondering if the content duplication from docs is because of a comment I'd made that I didn't think all the documentation for these repos should be on the docs site because some of it would be more readily located by developers if it was right in the repo?

I think it would be fine to link to the docs page for the sections that have to do with actions taken on the dashboard / anything not about wrangling the files inside a fork of this repo.

Bottom line I thought the docs in this particular repo were pretty much good as they were, except for the stuff that says experimental. But it's docs, I'm not sure how much my opinion should matter on it at the end of the day either.

**Comment by alexdicianu** [comment]
@mbaynton I'm unsure of what to put in the readme but, I admit that duplicating the docs is probably not a good idea (maintainability wise). Why not just have something simple like what we have now without the "experimental" part and just link to the docs page for which we have a process to edit and more people are empowered to do so?

**Review by mbaynton — APPROVED** [review]
This PR has been in my GitHub review requests for years and I don't seem to have another way to clear it.
